packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1287,10 +1287,12 @@ export default class Meeting extends StatelessWebexPlugin {
     password = null,
     captchaCode = null,
     extraParams = {},
+    hasPrejoinStarted = false,
```

Comment:
we need to update the jsdoc above with the new param, but this also makes me wonder what should we say in those docs for all the other apps that use JS-SDK, maybe we could avoid adding this param to the public API and know that we're in "pre-join" phase?